packages/apps/human-app/server/src/modules/jobs-discovery/jobs-discovery.controller.ts

```diff
@@ -41,6 +46,9 @@ export class JobsDiscoveryController {
         JobsDiscoveryParamsCommand,
       );
     jobsDiscoveryParamsCommand.token = token;
-    return await this.service.processJobsDiscovery(jobsDiscoveryParamsCommand);
+    return await this.service.processJobsDiscovery(
```

Comment:
Use JobsDiscoveryParamsCommand to pass the qualifications and follow the rules implemented in the whole application